docs/develop/create-addin-commands-unified-manifest.md

```diff
@@ -0,0 +1,601 @@
+                },
+                {
+                    "size": 80,
+                    "file": "https://localhost:3000/assets/icon-80.png"
+                }
+            ],
+            "controls": [
+                {
+                    // markup omitted 
+                }
+            ]
+        }
+    ]
+    ```
+
+1. Ensure that there is a control object in the "controls" array for each button or custom menu you want. The following is an example. Note the following about this markup:
+
+    - The "id", "label", and "icons" properties have the same purpose and the same restrictions as the corresponding properties of a group object, except that they apply to a specific button or menu within the group.
+    - The "type" property is set to "button" which means that the control will be a ribbon button. You can also configure a task pane command to be executed from a menu item. See [Menu and menu items](#menu-and-menu-items).
+    - The "supertip.title" (maximum length: 64 characters) and "supertip.description" (maximum length: 128 characters) appear when the cursor is hovering over the button or menu.
+    - The "actionId" must be an exact match for the "runtimes.actions.id" that you set in [Configure the runtime for the task pane command](#configure-the-runtime-for-the-task-pane-command).
+
+    ```json
+    {
+        "id": "msgReadOpenPaneButton",
+        "type": "button",
+        "label": "Show Task Pane",
+        "icons": [
+            {
+                "size": 16,
+                "file": "https://localhost:3000/assets/icon-16.png"
+            },
+            {
+                "size": 32,
+                "file": "https://localhost:3000/assets/icon-32.png"
+            },
+            {
+                "size": 80,
+                "file": "https://localhost:3000/assets/icon-80.png"
+            }
+        ],
+        "supertip": {
+            "title": "Show Contoso Task Pane",
+            "description": "Opens the Contoso task pane."
+        },
+        "actionId": "ShowTaskPane"
+    }
+    ```
+
+You have now completed adding a task pane command to your add-in.  [Sideload and test it](../testing/test-debug-office-add-ins.md#sideload-an-office-add-in-for-testing).
+
+## Add a function command
+
+The following subsections explain how to include a [function command](../design/add-in-commands.md#types-of-add-in-commands) in an add-in.
+
+### Create the code for the function command
+
+1. Ensure that your source code includes a JavaScript or Typescript file with the function that you want to execute with your function command. The following is an example. Since this article is about creating add-in commands, and not about teaching the Office JavaScript Library, we provide it with minimal comments, but do note the following:
+
+    - For purposes of this article, the file is named commands.js.
+    - The function will cause a small notification to appear on an open email message with the text "Action performed".
```

Comment:
Is it necessary to specify exactly where this notification occurs?
```suggestion
    - The function will cause a small notification to appear with the text "Action performed".
```

Reply:
> Is it necessary to specify exactly where this notification occurs?

I think so. It is very small and inconspicuous. The first time I ran the Outlook add-in created by Yo Office, I thought it was broken because I literally didn't notice notification.